### DISCUSSION TITLE
Patch 2305 manually

### DISCUSSION THREAD

**Discussion by XRedCubeX** [LSPosed/MagiskOnWSALocal#579] [Q&A] [answered]
Hi all, I'm trying to install and patch WSA 2305 but when I start the script it only finds 2304, I'm part of the WSA Preview program and I have access to version 2305 if I use the microsoft store, how do I manually download that version and I patch it?

**Comment by fromhgbwithluv** [comment]
@XRedCubeX You can use my script to make a custom build with v2305
https://github.com/YT-Advanced/WSA-Script

**Reply by XRedCubeX** [reply]
> @XRedCubeX You can use my script to make a custom build with v2305 https://github.com/YT-Advanced/WSA-Script

But now I would like to try to get the appx file manually by downloading it from my microsoft store, can this be done?

**Reply by fromhgbwithluv** [reply]
@XRedCubeX Yes, you can download it in my WSAPackage repos
Link: https://github.com/YT-Advanced/WSAPackage/releases/tag/v2305.40000.2.0

**Reply by XRedCubeX** [reply]
> @XRedCubeX Yes, you can download it in my WSAPackage repos
> Link: https://github.com/YT-Advanced/WSAPackage/releases/tag/v2305.40000.2.0

I mean a way to download it myself from the servers

**Reply by XRedCubeX** [reply]
> @XRedCubeX Yes, you can download it in my WSAPackage repos
> Link: https://github.com/YT-Advanced/WSAPackage/releases/tag/v2305.40000.2.0

I mean a way to download it myself from the servers

**Reply by fromhgbwithluv** [reply]
@XRedCubeX You need to join WSA Insider Preview Program, then get from it

**Reply by XRedCubeX** [reply]
> @XRedCubeX You need to join WSA Insider Preview Program, then get from it

Im part of it

**Reply by fromhgbwithluv** [reply]
So just use Microsoft Store and follow this guide (Fidler guide):
https://woshub.com/how-to-download-appx-installation-file-for-any-windows-store-app/

**Comment by Almighty188** [comment]
Read the following post:
![image](https://github.com/LSPosed/MagiskOnWSALocal/assets/78993393/a8592403-da82-4b28-bc98-a11154fb5686)

**Reply by XRedCubeX** [reply]
> Read the following post: ![image](https://user-images.githubusercontent.com/78993393/243193392-a8592403-da82-4b28-bc98-a11154fb5686.png)

I was looking for a way to download it myself

**Reply by Almighty188** [reply]
Yes, that includes how to first download it yourself.  I could never get MS Store to show WSA, only works when I install MagiskOnWSALocal.

**Reply by XRedCubeX** [reply]
> Yes, that includes how to first download it yourself. I could never get MS Store to show WSA, only works when I install MagiskOnWSALocal.

Instead, I have to click on a link that takes me to WSA for the microsoft store but if I look for it it doesn't appear

**Comment by XRedCubeX** [comment, marked as answer]
Solved by downloading it with MS Store and copying it manually from WindowsApps folder
